services/spar/migrate-data/deb/opt/spar/spar-migrate-data/spar-migrate-data

```diff
@@ -0,0 +1 @@
+../../../../../../../dist/spar-migrate-data
```

Comment:
Best hacks live forever :1st_place_medal: 

Can you please write about this somewhere?